### ISSUE TITLE
Update wiki page

### ISSUE THREAD

**Issue by vbjay** [ClosedXML/ClosedXML#335]
**Do you want to request a *feature* or report a *bug*?**
- [ ] Bug
- [x] Feature

**What is the current behavior?**
Wiki says you can't insert images. See https://github.com/ClosedXML/ClosedXML/wiki/How-can-I-insert-an-image%3F

**If the current behavior is a bug, please provide the steps to reproduce and 
if possible a minimal demo of the problem with a sample spreadsheet.**

**What is the expected behavior or new feature?**
Updated wiki page that describes image loading per #324 #323 
**Did this work in previous versions of our tool?  Which versions?**

- [ ] I attached a sample spreadsheet.  (You can drag files on to this issue)

**Code to reproduce problem:**
```c#
public void Main()
{
    // Where possible, post full code to reproduce your issue that adheres to:
    // - Fully runnable. I should be able to copy and paste this code into a 
    //   console application and run it without having to edit it much.
    // - Declare all your variables (this follows from the previous point)
    // - The code should be a minimal code sample to illustrate issue. The code 
    //   samples on the wiki are good examples of the terseness that I want. Don't
    //   post your full application.
}
```

**Comment by igitur** [comment]
a PR would be nice.

**Comment by vbjay** [comment]
I agree. However, the wiki is not editable by non members.

**Comment by igitur** [comment]
Oh I see. I think this approach is best: https://stackoverflow.com/a/11481887/179494

Anyway, I will merge it only once the images API is merged into the `master` branch.

**Comment by vbjay** [comment]
Sure. This was just a reminder to make sure the wiki didn't get out of
sync.

**Comment by igitur** [comment]
@vbjay If I give you privileges, would you help maintaining the wiki? I can't get to everything.
